plugins/atomicfu/atomicfu-runtime/build.gradle.kts

```diff
@@ -23,4 +24,12 @@ kotlin {
             }
         }
     }
-}
\ No newline at end of file
+}
+
```

Comment:
Please also add here before `publishing {`: `configureCommonPublicationSettingsForGradle()`.

This will add common Kotlin publication info.